client/src/setup/setupLanguageServerHooks.ts

```diff
@@ -78,15 +80,16 @@ const startLanguageServer = (extensionState: ExtensionState): void => {
 
   setupIndexingHooks(extensionState, client);
 
-  setupValidationJobHooks(extensionState, client)
+  await setupValidationJobHooks(extensionState, client)
     .then((disposable) => extensionState.listenerDisposables.push(disposable))
     .catch((err) => extensionState.logger.error(err));
 
   const notifyTelemetryChanged = () => {
     const telemetryEnabled = isTelemetryEnabled();
 
     extensionState.telemetryEnabled = telemetryEnabled;
-    client.sendNotification("custom/didChangeTelemetryEnabled", {
+
+    void client.sendNotification("custom/didChangeTelemetryEnabled", {
```

Comment:
As a follow up, I think we should have a helper that does this `void` trick, which should have docs about why it's done.